test/e2e/satellite_e2e_test.go

```
@@ -0,0 +1,313 @@
+	require.NotEmpty(t, token, "token should not be empty")
+
+	t.Logf("satellite registered successfully with token: %v", token)
+
+	s.pushImageToSourceRegistry(t)
+
+	//ZTR
+	configPath := fmt.Sprintf("%s/test/e2e/testdata/new_config.json", s.projectDir)
+
+	satelliteDir := s.client.Host().Directory(s.projectDir)
+
+	_, err = s.client.Container().
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)